src/screens/login/LoginScreen.js

```diff
@@ -231,6 +243,48 @@ const LoginSteps = ({
     }
   }
 
+  const startMagicLinkOAuth = useCallback(
+    async (fqdn, magicCode) => {
+      if (await NetService.isOffline())
+        NetService.handleOffline(routes.authenticate)
+
+      try {
+        const instance = 'https://' + fqdn
```

Comment:
This time we're not only working with a remote / local cloudery right. We can use this `startMagicLinkOAuth` with a localhost instance, right? If yes, we should add http if localhost or cozy.tools, no?

Reply:
We can try to guess it with localhost, cozy.tools and nip.io yes. Are you ok if I do this edit in another PR in the following days?